pep/app/services/exports.ts

```diff
@@ -87,17 +88,24 @@ export default class ExportsService extends Service {
             navigator.msSaveBlob(blob, filename);
             return;
         }
+        this._downloadItem(blob, filename, mimetype);
+    }
 
+    _downloadItem(urlOrBlob: string | Blob, filename: string, mimetype?: string) {
```

Comment:
why is this method named with the `_` "im private" convention when its used publicly outside of the class?

also missing docblock comment on this method